app/helpers/events_helper.rb

```diff
@@ -7,11 +7,11 @@ def event_row(event, &block)
     end
   end
 
-  def toggle_event_button(event)
+  def toggle_event_button(organization, event)
```

Comment:
Going back on my earlier suggestion. Sorry! We can figure out an event's organization from the event alone:

```ruby
# app/models/event.rb
class Event < ActiveRecord::Base
  # ...
  belongs_to :organization
  # ...
end
```

```ruby
def toggle_event_button(event)
  if event.active?
    button_to 'Deactivate', deactivate_organization_path(event.organization, event)
  else
    button_to 'Activate', activate_organization_event_path(event.organization, event)
  end
end
```